compile_js_i18n

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+for locale in pt-br; do
```

Comment:
We can probably get this from `locale/*` so we don't have to update this list manually.

Reply:
I just removed this file. We're already doing it using makefile.